scripts/tsbs-results-query.py

```diff
@@ -0,0 +1,85 @@
+import csv
+import re
+import json
+import os
+import time
+import psycopg2
+
+
+print('-=-=- BEGIN TSBS QUERY LOAD -=-=-');
+
+resultlist = list()
+with open('tsbsqueryoutput.txt', newline='') as csvf:
+    cread = csv.reader(csvf, delimiter=',', quotechar='"')
+    for row in cread:
+        print(len(row));
+        if len(row) > 0 and (re.compile('^Run').match(row[0]) or len(resultlist) > 0):
+            resultlist.append(row)
+
+print('-=-=-=-');
+for row in resultlist:
+    print(row);
+print('-=-=-=-');
+
+
+query = re.compile('[\w\s]+[\w]');
+print(query.search(resultlist[3][0]).group());
+
+entry = {};
+entry['type'] = query.search(resultlist[3][0]).group();
+entry['date'] = time.strftime("%a, %d %b %Y %H:%M:%S +0000", time.gmtime());
+#entry['repo'] = os.environ.get('GITHUB_REPOSITORY','missing_GITHUB_REPOSITORY');
+entry['repo'] = os.environ.get('DRM_GIT_REPO','missing_DRM_GIT_REPO');
+#entry['ref'] = os.environ.get('GITHUB_REF', 'missing_GITHUB_REF');
+entry['ref'] = os.environ.get('DRM_GIT_BRANCH', 'missing_DRM_GIT_BRANCH');
+#entry['commitid'] = os.environ.get('GITHUB_SHA', 'missing_GITHUB_SHA');
+entry['commitid'] = os.environ.get('DRM_GIT_COMMIT', 'missing_DRM_GIT_COMMIT');
+entry['cpuinfo'] = os.environ.get('CPU_INFO', 'missing_CPU_INFO');
+entry['date'] = time.strftime("%a, %d %b %Y %H:%M:%S +0000", time.gmtime());
+entry['entries'] = {};
```

Comment:
Not sure why you have these different variables, but some documentation of the module would solve that.